src/pages/blog/how-we-design-at-mintlify/index.mdx

```diff
@@ -0,0 +1,137 @@
+import { ArticleLayout } from '@/components/blog/ArticleLayout'
+
+export const meta = {
+  title: 'How We Approach Design At Mintlify',
+  description:
+    ': Good design is more than a process, it’s a headspace. Here are the building blocks of our approach to design. ',
+  mobileImageUrl: '/static/images/blog/august-2023/mobile.png',
+  desktopImageUrl: '/static/images/blog/august-2023/mobile.png',
+  author: 'Han Wang',
+  authorRole: 'Co-Founder',
+  date: 'November 14, 2023',
+  categories: ['Featured', 'Product'],
+  authorImageUrl: '/static/images/blog/authors/han.png',
+  readingTime: '2 minute read',
+}
+
+export default (props) => <ArticleLayout meta={meta} {...props} />
+
+<img
+  src="/static/images/blog/october-2023/desktop.png"
+  alt="blog thumbnail"
+  className="w-full rounded-md"
+/>
+
+When we ask our users why they moved to Mintlify, it’s often because of the user experience of our documentation. It’s something that we are always thinking about, and that we devote countless hours and iterations to.
+
+So what exactly makes good design? When you really boil it down, good design is all about taste. 
```

Comment:
Maybe add a GIF somewhere after this line or whenever relevant? Something that comes up about "taste" or "tasty" here maybe?

Reply:
Added GIF